### DISCUSSION TITLE
Do allowed_amounts include the participants' cost?

### DISCUSSION THREAD

**Discussion by mattgiguere** [CMSgov/price-transparency-guide#632] [Q&A] [answered]
The description of the allowed_amount field in the description of the [Out-Of-Network Payment Object](https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/allowed-amounts#out-of-network-payment-object) documentation currently says the following:

> The allowed amount must be reported as the actual dollar amount the plan or issuer paid to the out-of-network provider for a particular covered item or service, plus the participant’s, beneficiary’s, or enrollee’s share of the cost. See additional notes.

However, looking at the "Additional Notes" at the bottom of that section it doesn't mention anything about including the participant's, beneficiary's, or enrollee's share of the cost. 

My understanding of allowed amounts is that they represent the dollar amounts that issuers will pay towards and item or service and that the participant pays for the remainder, but the description seems to indicate that the allowed_amount here represents the COMBINED amount that the issuer paid PLUS what the participant paid. 

Can you please clarify what the allowed_amount here is supposed to represent?

**Comment by bryanbriegel** [comment, marked as answer]
If helpful, review here:  https://www.healthcare.gov/glossary/allowed-amount/   

Allowed is the total compensation allowed for a service by the plan.  This amount may be paid in full by the health plan, or partially paid by the health plan with a portion of this amount paid by the member.

**Comment by jltang-kp** [comment]
@bryanbriegel echo your points. The allowed amount is the compensation for the rendering provider. It can be a combination of participants cost as well as the health plan compensation. 

I would assume it would be unknown for figuring out the exact breakdown of cost from the participant unless the respective benefits are provided somewhere in the file.

**Comment by mattgiguere** [comment]
Ah, that makes sense. Thanks for your help! When the description says "plus the participant’s, beneficiary’s, or enrollee’s share of the cost" they are not talking about the participant's share of the cost that is beyond the allowed amount (and therefore not the full billed amount) they are only referring to the share of the cost that benefits did not cover. Seems like it would be helpful to edit the description to clarify that.
